src/Symfony/Component/Security/Core/Authentication/Provider/DaoAuthenticationProvider.php

```diff
@@ -108,23 +108,29 @@ protected function checkAuthentication(UserInterface $user, UsernamePasswordToke
     /**
      * {@inheritdoc}
      */
-    protected function retrieveUser(string $username, UsernamePasswordToken $token)
+    protected function retrieveUser(string $userIdentifier, UsernamePasswordToken $token)
```

Comment:
I think we can keep `$username` here as this class is only about `UsernamePasswordToken` tokens

Reply:
I'm not sure. I think the `UsernamePasswordToken` also implies "user identifier", but I don't think `UserIdentifierPasswordToken` is nice, nor is `UserPasswordToken` imho.

Reply:
Fine for me as long as psalm does not complain about it